src/libraries/Microsoft.PowerFx.Connectors/OpenApiExtensions.cs

```diff
@@ -337,10 +337,35 @@ internal static void WhenPresent(this OpenApiObject apiObj, string str, Action<O
                 action(openApiObj);
             }
         }
+        
+        private static int OpenApiParameterHash(this OpenApiParameter openApiParameter)

```

Comment:
>int [](http://example.com/codeflow?start=23&length=3)

From a _correctness_ point of view, would everything still work if this always returned 0?

Reply:
Absolutely not, it's a hash
But if it would use a static i++, that would work, yes